specification/securityinsights/resource-manager/readme.md

```diff
@@ -54,6 +54,10 @@ input-file:
   - Microsoft.SecurityInsights/stable/2024-09-01/SourceControls.json
   - Microsoft.SecurityInsights/stable/2024-09-01/ThreatIntelligence.json
   - Microsoft.SecurityInsights/stable/2024-09-01/Watchlists.json
+suppressions:
+  - code: AvoidAdditionalProperties
+    from: dataConnectors.json
```

Comment:
pl book an api review or modelling office hours to discuss the scenario..

Reply:
booked a review.
although this exact suppresion is shown in the private version - https://statics.teams.cdn.office.net/evergreen-assets/safelinks/1/atp-safelinks.html
the reviewer asked that we had a where clause